_config.yml

```diff
@@ -52,7 +52,8 @@ links:
 # http://en.wikipedia.org/wiki/List_of_tz_database_time_zones
 timezone:    Asia/Kolkata
 future:      true
-highlighter: pygments
+
```

Comment:
Remove space

Reply:
I added a line to separate out the code snippet features that is markdown and highlighter.